### ISSUE TITLE
Pudge Arcana Hook Icon Bug

### ISSUE THREAD

**Issue by pkay01x** [ValveSoftware/Dota2-Gameplay#5253]
### Cosmetic Item Name

The Abscesserator  And Pudge Arcana

### Description

Pudge Arcana Icon Doesn't Let Me Change Hook Icon when i use The Abscesserator , 
When i dont use arcana the hook icon displays of The Abscesserator but when i equip arcana it disappears automatically  ! 

### Example Match ID

no need Match ID happens in Loadout 

### Screenshots

![image](https://user-images.githubusercontent.com/117821522/200809956-6e0191e3-4f12-4631-95bf-b15d5b945c4f.png)

**Comment by Debeluk** [comment]
![image](https://user-images.githubusercontent.com/117806927/200819733-7e1ece74-9d23-4f12-9e2d-48fe6db1bec0.png)
choose the Abscesserator on the meat hook, like in my screenshot, should work

**Comment by pkay01x** [comment]
> 
it doesn't , 
Its already equipped but doesn't work with arcana , works with other normal skins though. 
![image](https://user-images.githubusercontent.com/117821522/201018684-d415963d-1488-44d8-9620-d65267f347eb.png)

**Comment by Debeluk** [comment]
> > 
> 
> it doesn't , Its already equipped but doesn't work with arcana , works with other normal skins though. ![image](https://user-images.githubusercontent.com/117821522/201018684-d415963d-1488-44d8-9620-d65267f347eb.png)

have you tried changing styles ?

**Comment by pkay01x** [comment]
> 

yes sir i click and nothing happens , after i unequip Arcana only its visible

**Comment by pkay01x** [comment]
![image](https://user-images.githubusercontent.com/117821522/201097685-64bfb3bd-338a-4f81-950e-e5f108ae352b.png)

**Comment by SerGlushko** [comment]
The issue is that the arcana takes complete control over your icons and isn't displayed as an option in the select menu at all
